.buildkite/build-pipeline.yml

```diff
@@ -1 +1,30 @@
-# Intentional empty definition because it is in 8.x
\ No newline at end of file
+- label: "Build pipeline"
```

Comment:
review note: specific branch build job runs when PR gets merged.
FYI: periodic build job runs every branch, scripts will be placed on plugin main branch.